concent_api/core/message_handlers.py

```diff
@@ -280,7 +324,10 @@ def handle_send_reject_report_computed_task(client_message):
 
     if get_current_utc_timestamp() <= deserialized_message.compute_task_def['deadline'] + settings.CONCENT_MESSAGING_TIME:
         if subtask.ack_report_computed_task_id is not None or subtask.ack_report_computed_task_id is not None:
-            raise Http400("Received RejectReportComputedTask but AckReportComputedTask or another RejectReportComputedTask for this task has already been submitted.")
+            raise Http400(
+                "Received RejectReportComputedTask but AckReportComputedTask or another RejectReportComputedTask for this task has already been submitted.",
+                error_code=ErrorCode.QUEUE_RESPONSE_ALREADY_SUBMITTED,
```

Comment:
@cameel Should it be `SUBTASK_DUPLICATE_REQUEST` too ?

Reply:
Yeah. I generally tried not to add a comment in every place I found because there would be a lot of repetition. If I mentioned something in one place it's very possible that there are other places that are like this.